tests/backup/backup_helper.go

```diff
@@ -1048,24 +1036,18 @@ func generateEncryptionKey() string {
 	return string(b)
 }
 
-func GetScheduleUID(scheduleName string, orgID string, ctx context.Context) (string, error) {
+func GetScheduleUID(backupScheduleName string, orgID string, ctx context.Context) (string, error) {
 	backupDriver := Inst().Backup
-	bkpUid, err := backupDriver.GetBackupUID(ctx, scheduleName, orgID)
-	if err != nil {
-		return "", fmt.Errorf("failed to get backup UID for %s, Err: %v", scheduleName, err)
-	}
 	backupSchInspectRequest := &api.BackupScheduleInspectRequest{
```

Comment:
Change it from `backupSchInspectRequest` to `backupScheduleInspectRequest`

Reply:
Done.

Reply:
I dont see this